<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStickerView</class>
 <widget class="QDialog" name="DialogStickerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="LBL_DLG_STICKER_VIEW_TITLE">View Sticker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lh3.qrc">
    <normaloff>:/lh3/Resources/operators_stickers.png</normaloff>:/lh3/Resources/operators_stickers.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>9</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lCaptionTitle">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string comment="LBL_DLG_STICKER_VIEW_TITLE_L">Title:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lTitle">
       <property name="text">
        <string>The test message</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lCpationAuthor">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string comment="LBL_DLG_STICKER_VIEW_AUTHOR">Author:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="lAuthor">
       <property name="text">
        <string>demo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lCaptionDate">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string comment="LBL_DLG_STICKER_VIEW_DATE">Date:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lDate">
       <property name="text">
        <string>10.06.2009 10:51</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="teMessage">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbMarkAsUnread">
       <property name="text">
        <string comment="LBL_DLG_STICKER_VIEW_MARK_UNREAD">Mark as unread</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string comment="LBL_CLOSE">Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lh3.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogStickerView</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
